src/main/java/net/logstash/logback/composite/loggingevent/MessageJsonProvider.java

```diff
@@ -27,18 +28,41 @@
     
     public static final String FIELD_MESSAGE = "message";
 
+    private Pattern lineSeparatorPattern = null;
+    private boolean writeMultiLineMessage = false;
```

Comment:
Same comment as LogstashFormatter regarding naming of these fields.  Combine into a single "messageSplitRegex" field.

Also add some javadoc for this field.